test/quantized_ops/test_dot_general.py

```diff
@@ -56,6 +56,13 @@ def test_dot_general_int32_dtype(self):
         preferred_element_type=torch.int32)
     self.assertTrue(torch.allclose(xla_out.cpu(), expected_out))
 
+  def test_raises_error_on_non_xla_tensor(self):
+    lhs = torch.rand(10, 3, 4, dtype=torch.bfloat16)
+    rhs = torch.rand(10, 4, 5, dtype=torch.bfloat16, device="xla")
+    error_message = r"Expected input tensor `lhs` to be an actual XLA tensor. Got: CPUBFloat16Type"
```

Comment:
The error message shouldn't reference the name `lhs` as in the actual user code the parameter name may be different. Just say "the first argument".

Reply:
My reasoning for using `lhs` was that it's a formal parameter for `_xla_dot_general()`. When we call `help()` on it, or when it's called with mismatching parameter types, `lhs` and `rhs` names are there:

```python
>>> torch_xla._XLAC._xla_dot_general(5, 3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: _xla_dot_general(): incompatible function arguments. The following argument types are supported:
    1. (lhs: torch.Tensor, rhs: torch.Tensor, dimension_numbers: tuple, precision_config: Optional[str] = None, preferred_element_type: Optional[object] = None) -> torch.Tensor

Invoked with: 5, 3
```

Do you think we should replace it by "the first argument"?

Reply:
I would prefer "the first argument" as it's clear by itself. If we write "lhs", the user would need to look up the documentation/help for `_xla_dot_general` to know what we are talking about.